src/Microsoft.IdentityModel.Tokens/ECDsaAdapter.cs

```diff
@@ -62,9 +62,6 @@ internal ECDsaAdapter()
                 CreateECDsaFunction = CreateECDsaUsingECParams;
             else
                 CreateECDsaFunction = CreateECDsaUsingCNGKey;
-#elif NETSTANDARD1_4
-            if (SupportsCNGKey())
-                CreateECDsaFunction = CreateECDsaUsingCNGKey;
 #elif DESKTOP
```

Comment:
I opened up this issue to address how we deal with DESKTOP
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1325